config/initializers/application_controller_renderer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+
+# ActiveSupport::Reloader.to_prepare do
+#   ApplicationController.renderer.defaults.merge!(
+#     http_host: 'example.org',
+#     https: false
+#   )
+# end
```

Comment:
Do these initializers come with a rails upgrade or something?

Reply:
Yep, rails provides an app:update tool that adds in (or updates) the various initialisers. I tend to keep them around, even when essentially empty, as they are useful documentation.